### DISCUSSION TITLE
hideCloseButton not working

### DISCUSSION THREAD

**Discussion by nmarriotti** [gilbarbara/react-joyride#1012] [Q&A]
I'm using the latest release version 2.7.4 and the hideCloseButton property of both the steps and Joyride component are not working. How can I hide the close button?

**Update**
The hideCloseButton property looks to be controlling the X that appears in the top-right corner of the tooltip and not the Close button in the tooltip footer.

**Comment by gilbarbara** [comment]
Hey @nmarriotti 
Yes, the "X" is the close button.

**Reply by gilbarbara** [reply]
I'd assume you're setting the `continuous` prop to false.
How would people close the tooltip if you hid the close button in the footer?
Using the X button?

**Comment by tech-jw** [comment]
For anyone else, just set `hideFooter: true` in the step you want to hide the close button in
